src/bundles/peer-locations.js

```diff
@@ -250,26 +275,23 @@ class PeerLocationResolver {
   }
 
   optimizedPeerSet (peers) {
-    if (this.pass && this.pass < 3) {
+    if (this.pass < 3) {
       // use a copy of peers sorted by latency so we can resolve closest ones first
       // (https://github.com/ipfs-shipyard/ipfs-webui/issues/1273)
       const ms = x => (parseLatency(x.latency) || 9999)
```

Comment:
can we avoid doing the parse inside the sort ?

Reply:
We could, but that requires another selector dependency just to parseLatency for each peer. And you also just need to parse the latency if it's the first 3 poller requests